### PR TITLE
Extract logic for managing dashboard query ranges to hook

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/use-duration-hook.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  [Duration.SIX_HR]: 6 * ONE_HOUR,
+  [Duration.TWENTY_FOUR_HR]: 24 * ONE_HOUR,
+};
+
+export const useMetricDuration = (): MetricDuration => {
+  const [duration, setDuration] = React.useState(Duration.ONE_HR);
+  const setMetricDuration = React.useCallback((d: Duration) => setDuration(Duration[d]), [
+    setDuration,
+  ]);
+  return [duration, setMetricDuration, Duration];
```
Comment (mareklibra) [inline frontend/packages/console-shared/src/components/dashboard/use-duration-hook.ts]: WHy do we need to return the `Duration` when it is a constant?

Comment (rawagner) [reply frontend/packages/console-shared/src/components/dashboard/use-duration-hook.ts]: not really needed, removed

**Pull request merged**